docker/conf/docker-compose-mac.yml.dist

```diff
@@ -28,7 +28,7 @@ services:
         depends_on:
             - php-fpm
         volumes:
-            - shopsys-framework-web-sync:/var/www/html/project-base/web
+            - ./project-base/web:/var/www/html/project-base/web:cached
```

Comment:
If synced `./project-base/web` will do fine it could be replaced in `docker-compose.yml.dist` for webserver too.

Reply:
On Linux it does not matter what folder do you wanna create volume from as actually there are no volumes created on linux

Reply:
Yes there are.